### DISCUSSION TITLE
Upgrade error

### DISCUSSION THREAD

**Discussion by samuellhwarren** [docker-library/ghost#410] [General]
when trying to update from 5.75-alpine I get the following error:
failed to pull images of the stack: db Pulling ghost Pulling db Pulled no matching manifest for linux/arm64/v8 in the manifest list entries

I am running headless aarch64 on raspberry pi. Any suggestions for updates? or

**Comment by tianon** [comment]
I'd suggest providing more detail about what you've done -- I can't reproduce a failure pulling the `arm64` build of any of the supported tags of `ghost` :sweat_smile:
